content/docs/quickstart/kubernetes/_index.md

```diff
@@ -6,12 +6,23 @@ menu:
     identifier: kubernetes
     name: Kubernetes
     weight: 2
+cloudlink: /docs/reference/clouds/kubernetes
 ---
 
-This guide is intended to help you quickly get started provisioning infrastructure on Kubernetes with Pulumi.
+Pulumi's Cloud Native SDK makes it easy to target any Kubernetes environment to
+provision a cluster, configure and deploy applications, and update them as
+required.
 
-<!-- TODO embellish the summary/overview of the guide. -->
+Pulumi supports programming against Kubernetes---Minikube, on-premises and
+cloud-hosted custom Kubernetes clusters, and the managed services from Google
+(GKE), Azure (AKS), and Amazon (EKS). Pulumi's [Kubernetes provider
```

Comment:
Wondering if we should cut the last sentence, to avoid linking elsewhere to encourage forward progress on the guide.

If we keep it, the CLI link should use `relref`.